peachjam/templates/peachjam/saved_document/_folders_list.html

```diff
@@ -15,15 +15,56 @@ <h2>{% trans 'Saved documents' %}</h2>
         <h5 class="card-header">
           <i class="bi bi-folder"></i> {{ folder.name }}
         </h5>
+        <div class="card-header">
+          <ul class="nav nav-tabs card-header-tabs">
+            <li class="nav-item">
+              <button class="nav-link active"
+                      id="saved-doc-tab-{{ folder.pk }}"
+                      data-bs-toggle="tab"
+                      data-bs-target="#saved-docs-{{ folder.pk }}"
+                      type="button"
+                      role="tab"
+                      aria-controls="saved-doc-tab-{{ folder.pk }}"
```

Comment:
this and the data-bs-target should be the same element AFAIK

Reply:
done